sw/otbn/crypto/BUILD

```diff
@@ -6,11 +6,230 @@ load("//rules:otbn.bzl", "otbn_binary", "otbn_library")
 
 package(default_visibility = ["//visibility:public"])
 
+otbn_library(
+    name = "modexp",
+    srcs = [
+        "modexp.s",
+    ],
+)
+
+otbn_library(
+    name = "p256",
+    srcs = [
+        "p256.s",
+    ],
+)
+
+otbn_binary(
+    name = "p256_base_mult_test",
+    srcs = [
+        "p256_base_mult_test.s",
+    ],
+    deps = [
+        ":p256",
+    ],
+)
```

Comment:
If all of the `*_test` binaries are really tests, we should create a way to automatically execute these tests (including verifying the result) and encode that into an `otbn_test` rule.

It would be ok to do that in a follow-up PR.